edge-modules/MetricsCollector/Configuration.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace MetricsCollector
```

Comment:
I'm going to explore merging this with Settings.cs

Reply:
1.Merging with `Settings.cs`  does not quite make sense to me. 
This is more assimilate with `ModuleMessagesReport` and `ResponseOrientedReport` since they are simply just another report type (soon-to-be LogType).

2.  Can we consolidate `\iotedge\edge-modules\MessagesAnalyzer\ModuleMessagesReport.cs` and `\iotedge\edge-modules\MessagesAnalyzer\ModuleReport.cs`?

Reply:
1 - Aren't these just settings? Settings.cs sets configuration based on env vars. This Configuration.cs class sets configuration based on the module twin's desired properties. I agree that merging is not the best strategy, so I was thinking of renaming Configuration.cs to something more descriptive that indicates it sources settings from the module twin.

2 - I couldn't find ModuleReport. Are you sure this is the correct name, or am I looking in the wrong place?

Reply:
1. You're right now that I read the rest of the code. I don't think you should merge it though b/c you still want to be able to deploy MetricCollector without Analyzer. 
2. Interesting. Maybe my git is merging some branch together that I didn't notice.

Reply:
1 - The metrics collector also has a Settings.cs, which is what I was referring to